index.bs

```diff
@@ -287,32 +287,85 @@ mechanism for metadata discovery.
-unverified (such as indicating to a user that a display name of an
+If a listening agent ("controller") wants to connect to an advertising agent
+("receiver"), or to learn further metadata about it, it initiates a [[QUIC]]
+connection to the IP and port from the SRV record.  Prior to authentication,
+a message may be exchanged (such as further metadata), but such info should be
+treated as unverified (such as indicating to a user that a display name of an
 unauthenticated agent is unverified).
 
-To learn further metadata, an agent may send an agent-info-request
-message (see [[#appendix-a]]) and receive back an agent-info-response message.  The
-messages may contain the following information with the following meaning:
+To learn further metadata, the controller may send an agent-info-request
+message (see [[#appendix-a]]) and receive back an agent-info-response message.
+The messages may contain the following properties:
 
 : display-name (required)
-:: The display name of the responding agent intended
-    to be displayed to a user by the requesting agent.  If the responding agent
-    is not yet authenticated, the requesting agent should make UI affordance for
-    indicating to the user that the display name is not yet verified.  If the
-    responding agent changes its display name, the requesting agent should
-    make UI affordance for indicating to the user that the display name has
-    changed.
+:: The display name of the receiver, intended to be displayed to a user by the
+     controller. The controller should indicate through the UI if the receiver
+     is not authenticated or if the display name changes.
 
 : model-name (optional)
-:: If the agent is a hardware device, the model name of
+:: If the receiver is a hardware device, the model name of
     the device.  This is used mainly for debugging purposes, but may be
     displayed to the user of the requesting agent.
 
-<!-- TODO: Add device type and/or capabilities -->
+: audio-only (optional)
+:: Generally speaking, receivers are assumed to support streaming both
+    audio and video. If a receiver wishes to indicate that they only support audio
+    stream, they can pass this property as true. In that case, the controller
+    should indicate through the UI that the receiver is audio only.
+
+If the receiver is already authenticated, the controller has the ability to
+request additional information by sending an streaming-capabilities-request
+message, and receive back a streaming-capabilities-response message with the
+following properties:
+
+: max-pixel-fill-rate (required)
+:: The receiver shall reply with the supported pixel fill rate in megapixels per
+    second. Requesting agents can use this information to make better-educated
+    decisions about what resolution and frame rates to stream content at.
+
+: supported-codecs (required)
+:: The receiver must indicate to the controller what codecs it understands for
+    remote playback.
+
+: min-resolution (optional)
```

Comment:
+1  I'm not sure codecs belong in a device-level metadata, but instead should be part the negotiation specifically for streaming or remote playback.  Ideally we can share some messages between the two because there's overlap in the use case: the controller has some media it wants to play on the receiver with a given codec, resolution/framerate, and bitrate.  The receiver can answer if it can play that media at all and/or play it smoothly or efficiently (see the [Media Capabilities API](https://wicg.github.io/media-capabilities/) for a Web API for querying this data).